### ISSUE TITLE
Single Site Mode - URLs

### ISSUE THREAD

**Issue by bhandarys** [silexlabs/Silex#1170]
Hi, Does the URL for assets change in Single Site Mode? In Multi site mode all assets (images) were loaded in the template without any error. But in Single site mode, I get HTTP 404 error. One example is provided below

http://localhost:6805/ce/custom-service/get/src/libs/templates/silex-templates/air/assets/book-5.png [[HTTP/1.1 404 Not Found 1751ms]]()

However this works fine on Multi Site mode. Kindly help.

My versions are:
Node: 14.16.1
"silex-website-builder": "^2.7.29",
OS: Windows 10
Browser: Chrome (version 98.0.4758.102)

**Comment by lexoyo** [comment]
what is the path of the website, as shown in the network?

**Comment by bhandarys** [comment]
Thanks Alex for the prompt response. The path of the website is 

http://localhost:6805/?path=/src/techntour/websites/user1/editable.html&service=custom-service (It's on my local machine)

**Comment by lynnmedia** [comment]
REMOVE ME FROM YOUR EMAIL LIST BEFORE I DESTROY YOU!

**Comment by lynnmedia** [comment]
fuck you, die asshole

**Comment by lexoyo** [comment]
I will, eventually

I have reported your messages

Have a good day

**Comment by lexoyo** [comment]
> Thanks Alex for the prompt response. The path of the website is 
> 
> http://localhost:6805/?path=/src/techntour/websites/user1/editable.html&service=custom-service (It's on my local machine)

So the path of the assets is wrong. Please do this :

1. In multi site mode, start from a template, save the template in your new service
2. In single site mode open this new website

**Comment by bhandarys** [comment]
I completely missed this point. In multi site mode, when I saved the site for 3 users, user1, user2 and user3, only editable.html and editable,json was saved in the respective directory. Assets, CSS and JS folders were missing. I used Save Website / Custom Service and clicked OK on the top right where the suggested name was editable.html. Should I be doing something more to save the other folders and files?
